js/ui/popup.js

```diff
@@ -662,28 +663,42 @@ var Popup = Overlay.inherit({
     _setContentHeight: function() {
         (this.option("forceApplyBindings") || noop)();
 
-        if(this._disallowUpdateContentHeight()) {
-            return;
-        }
+        var toolbarsAndPaddingsHeight = this._getToolbarsAndPaddingsHeight(),
+            content = this._$content.get(0),
+            cssStyles = {};
 
-        var contentPaddings = this._$content.outerHeight() - this._$content.height(),
-            contentHeight = this._$content.get(0).getBoundingClientRect().height - contentPaddings;
-        if(this._$title && this._$title.is(":visible")) {
-            contentHeight -= this._$title.get(0).getBoundingClientRect().height || 0;
-        }
-        if(this._$bottom && this._$bottom.is(":visible")) {
-            contentHeight -= this._$bottom.get(0).getBoundingClientRect().height || 0;
+        if(this._isAutoHeight()) {
+            var container = $(this._getContainer()).get(0),
+                maxHeightValue = sizeUtils.calculateMaxHeight(this._getOptionValue("maxHeight", content), container, -toolbarsAndPaddingsHeight),
+                minHeightValue = sizeUtils.calculateMinHeight(this._getOptionValue("minHeight", content), container, -toolbarsAndPaddingsHeight);
+
+            if(minHeightValue !== undefined) {
+                cssStyles.minHeight = minHeightValue;
```

Comment:
```
extend(cssStyles, {
  minHeight: minHeightValue,
  maxHeight: maxHeightValue
})
```
In case value is `undefined` - cssStyles will not extend